workflows/storage/copy-restore.yaml

```
@@ -0,0 +1,142 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/argoproj/argo-workflows/v3.5.5/api/jsonschema/schema.json
+
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: copy-restore
+spec:
+  parallelism: 50
+  nodeSelector:
+    karpenter.sh/capacity-type: 'spot'
+  entrypoint: main
+  onExit: exit-handler
+  podMetadata:
+    labels:
+      linz.govt.nz/category: storage
+  arguments:
+    parameters:
+      - name: version_argo_tasks
+        description: 'Specify a version of the argo-tasks image to use, e.g. "v4.1" or "latest"'
+        value: 'v4'
+      - name: reports_location
+        description: 'Location of the archive restore report manifest files'
+        value: 's3://linz-workflows-scratch/restore/requests/'
+
+  templates:
+    - name: main
+      retryStrategy:
+        expression: 'false'
+      inputs:
+        parameters:
+          - name: version_argo_tasks
+            value: '{{workflow.parameters.version_argo_tasks}}'
+            default: 'v4'
+          - name: reports_location
+            value: '{{workflow.parameters.reports_location}}'
+            default: 's3://linz-workflows-scratch/restore/requests/'
+      dag:
+        tasks:
+          - name: list-manifests
+            templateRef:
+              name: tpl-at-list
+              template: main
+            arguments:
+              parameters:
+                - name: version_argo_tasks
+                  value: '{{inputs.parameters.version_argo_tasks}}'
+                - name: location
+                  value: '{{inputs.parameters.reports_location}}'
+                - name: include
+                  value: 'manifest.json$'
```

match whole filename?
```suggestion
                  value: '/manifest.json$'
```